project/src/main/java/com/google/sps/data/Restaurant.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+import java.util.List;
+import java.util.Map;
+
+@AutoValue
+public abstract class Restaurant {
+  public abstract String getName();
+  public abstract String getAddress();
+  public abstract Map<String, Double> getLatLngCoords();
```

Comment:
what does the string and double represent here? is it stringified location and then one of the coordinates?

Reply:
so the map looks like {"lat": 12.34, "lng": 56.78}. It should make it easier for plotting it with the Maps API. Should I indicate this?

Reply:
aaah I see , makes sense , yeah I think a comment above this that says. 
/** Example map: "lat" : 12, "lng" : 55 */ would help